registrations/models.py

```diff
@@ -241,7 +241,10 @@ class WebStoreMerchant(CreatedModifiedBaseModel):
         verbose_name=_("Phone number"),
         max_length=18,
     )
-    url = models.URLField(verbose_name=_("URL"))
+    url = models.URLField(
```

Comment:
Do we have any use of this field? Since its always set to `LINKED_EVENTS_UI_URL` it then could (and should) be removed.

Reply:
If the value of `settings.LINKED_EVENTS_UI_URL` is changed, then without this field we would need to check against the Talpa API if the value has changed to determine if a merchant needs to be updated. When saving to the local DB, we can check against that without an API request.